__tests__/utils.test.js

```diff
@@ -1,5 +1,6 @@
 import { expect } from 'chai';
-import { buildRegExpFromDelimiters } from '../src/components/utils';
+import { buildRegExpFromDelimiters, canDrag, canDrop } from '../src/components/utils';
+import noop from 'lodash/noop';
```

Comment:
can you move `noop` import to line 2 and give a new line after it to separate the import of utils